src/main/java/org/b612foundation/adam/datamodel/RunDescription.java

```diff
@@ -0,0 +1,124 @@
+package org.b612foundation.adam.datamodel;
+
+import java.util.Objects;
+
+import org.b612foundation.adam.opm.OrbitParameterMessage;
```

Comment:
I suppose at some point we'll pull this in? Alternatively we could keep opms as strings in Run objects to mirror Batch objects.

Reply:
Oh wait, obviously that's already here, pulled over with the other opm stuff. Nevermind!